hack/common.sh

```diff
@@ -44,7 +44,7 @@ NOOBAA_CSV="$OUTDIR_TEMPLATES/noobaa-csv.yaml"
 ROOK_CSV="$OUTDIR_TEMPLATES/rook-csv.yaml.in"
 OCS_CSV="$OUTDIR_TEMPLATES/ocs-operator.csv.yaml.in"
 
-LATEST_ROOK_IMAGE="rook/ceph:v1.8.3"
+LATEST_ROOK_IMAGE="subham03/rook:down-4.10"
```

Comment:
```suggestion
LATEST_ROOK_IMAGE="subham03/rook:down-4.10"
```
will change this later, before pr merges